pdns/bindparser.yy

```diff
@@ -65,6 +65,9 @@ void BindParser::parse(const string &fname)
 void BindParser::setDirectory(const string &dir)
 {
 	d_dir=dir;
+	if (d_dir.back() == '/') {
```

Comment:
this change and the associated test are because `//` is ugly. It isn't strictly part of this PR but it's annoying.